pom.xml

```diff
@@ -25,6 +25,8 @@
     <feign-okhttp.version>10.10.1</feign-okhttp.version>
     <!-- -->
     <tenant.yaml.file>${project.basedir}/src/main/resources/swagger.api/tenant.yaml</tenant.yaml.file>
+    <lombok.version>1.18.16</lombok.version>
```

Comment:
Just interesting why use this version instead of provided by `spring-boot-starter-parent`

Reply:
The version from parent pom is not compatible with intellij idea 2020.3 (see https://github.com/mplushnikov/lombok-intellij-plugin/issues/988)